src/lib/constants.ts

```diff
@@ -190,6 +190,56 @@ export const scopes: {
         description: "Access to execute your project's functions",
         category: 'Functions'
     },
+    {
+        scope: 'targets.read',
+        description: "Access to read your project's targets",
+        category: 'Messaging'
+    },
+    {
+        scope: 'targets.write',
+        description: "Access to create, update, and delete your project's targets",
```

Comment:
Below we call providers `messaging providers`. We should call those `messaging targets` then